Mathlib/AlgebraicGeometry/FunctionField.lean

```diff
@@ -125,7 +125,7 @@ theorem genericPoint_eq_bot_of_affine (R : CommRingCat) [IsDomain R] :
 
 instance functionField_isFractionRing_of_affine (R : CommRingCat.{u}) [IsDomain R] :
     IsFractionRing R (Scheme.Spec.obj <| op R).functionField := by
-  convert StructureSheaf.IsLocalization.to_stalk R (genericPoint _)
+  convert StructureSheaf.IsLocalization.to_stalk R (genericPoint (Scheme.Spec.obj (op R)))
```

Comment:
nightly-2024-05-19 and onwards need a bit more of a hint here, after the bugfix in https://github.com/leanprover/lean4/pull/4206